tests/unit/sca/conftest.py

```diff
@@ -112,26 +110,6 @@ def clean_directory(path: Path):
             os.remove(filepath)
 
 
-def fix_py37_win_tempdir_permissions(dirpath: Union[str, Path]) -> None:
```

Comment:
Why removing this ?

Reply:
remove fix for python 3.7, whose support was just dropped (https://github.com/GitGuardian/ggshield/pull/613)